djaoapp/templates/saas/billing/coupons.html

```diff
@@ -48,19 +48,14 @@
         <div class="col-md-3">
           <!-- Sort Coupons -->
           <div class="pb-3">
-            <div class="input-group input-group-md">
-              <div class="input-group-prepend">
-                <span class="input-group-text">{% trans %}Sort{% endtrans %}</span>
-              </div>
-              <select id="order-by" class="form-control" style="-webkit-appearance: none;">
-                <option value="code">{% trans %}Code (ascending){% endtrans %}</option>
-                <option value="-code">{% trans %}Code (descending){% endtrans %}</option>
-                <option value="percent">{% trans %}Discount (highest){% endtrans %}</option>
-                <option value="-percent">{% trans %}Discount (lowest){% endtrans %}</option>
-                <option value="ends_at">{% trans %}Expiration (soonest){% endtrans %}</option>
-                <option value="-ends_at">{% trans %}Expiration (latest){% endtrans %}</option>
-              </select>
-            </div>
+            <uiv-dropdown class="dropdown w-100" append-to-body>
```

Comment:
Unfortunately we cannot use `uiv` any longer because it only supports Bootstrap 3. That's why the `<select>` code was introduced in commit 972b56e1.